bpf/init.sh

```diff
@@ -260,12 +260,12 @@ function bpf_load()
 	bpf_compile $IN $OUT obj "$OPTS"
 	tc qdisc replace dev $DEV clsact || true
 	[ -z "$(tc filter show dev $DEV $WHERE | grep -v 'pref 1 bpf chain 0 $\|pref 1 bpf chain 0 handle 0x1')" ] || tc filter del dev $DEV $WHERE
-	cilium-map-migrate -s $OUT
+        cilium map migrate -s $OUT
 	set +e
 	tc filter replace dev $DEV $WHERE prio 1 handle 1 bpf da obj $OUT sec $SEC
 	RETCODE=$?
 	set -e
-	cilium-map-migrate -e $OUT -r $RETCODE
+	cilium map migrate -e $OUT -r $RETCODE
```

Comment:
Could we call `cilium map migrate` with the correct parameters instead of simply passing `$?` into a Go program? :/ This makes for spaghetti downstream.

Reply:
My take is that we will be getting rid of this anyways. ASAP. This PR is only to ensure that the core migrate logic works as intended.

Reply:
My take is that we will be getting rid of this anyways. ASAP. This PR is only to ensure that the core migrate logic works as intended.